host-bin/enter-chroot

```diff
@@ -684,10 +684,16 @@ if [ -n "$NOLOGIN" ]; then
         fi
     fi
 else
-    # Run rc.local
+    # Check and run rc.local
     if [ -n "$firstrun" -a -x "$CHROOT/etc/rc.local" ]; then
+        ( sleep 10;echo "Still running chroot's rc.local..." 2>&1 ) &
```

Comment:
@dnschneid Is it good in this way? Sorry for being not good at coding style and thanks for helping me correct it :)